### PR TITLE
Update v9 references to v10

### PR DESCRIPTION
**Pull request by vfonic**
I also added some minor changes. Let me know if you think those should be changed in any way or are not needed.

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/980)

### CODE REVIEW

**Comment by justin808** [comment]
Thanks @vfonic!

File: docs/tutorial.md [hunk 1]
```diff
@@ -52,23 +49,22 @@ rails new test-react-on-rails --webpack=react
...
 bundle exec rails webpacker:install:react
 ```
 
-
-
 Add the **React On Rails** gem to your Gemfile:
 
 ```
-gem 'react_on_rails', '9.0.0'         # use latest gem version, prefer exact version 
+gem 'react_on_rails', '10.0.2'         # prefer exact version
```
Comment (6temes) [inline docs/tutorial.md:62]: `# prefer exact gem version to match npm version`

It seems like a good idea to explain the reason.

Comment (vfonic) [reply docs/tutorial.md:62]: Indeed it does! Thanks! I'll create another PR. I also spotted one more sentence I could improve.

Comment (vfonic) [reply docs/tutorial.md:62]: https://github.com/shakacode/react_on_rails/pull/981 Clarify tutorial scope; Explain exact gem version

**Pull request merged**